_configs/eval_ssm.yaml

```diff
@@ -68,6 +68,19 @@ infmax_models:
         max_nb_clusters: 10
         experiment_name: "My Experiment"
         visualise: False
+  - name: "GBIM"
+    class: "GBIM"
+    parameters:
+      rng_seed: 43
+      device: cpu
+      common:
+        nb_seeds: 5
+        graphsample: 1000
+      datasample:
+        sample_num: 1000
+        simulate_times: 10
+      executor:
+        epochs: 300
```

Comment:
[2/3] if some of these parameters can be read from the global config, please revise their values

Reply:
I do not think that more of them would be used in other models. If it turns out otherwise, we should transfer it on an ongoing basis